static/styles/left-sidebar.scss

```diff
@@ -160,6 +166,7 @@ li.active-sub-filter {
     display: inline-block;
     min-width: $left_col_size;
     text-align: center;
+    margin-right: 3px;
```

Comment:
I think we want there to be less space between the names and the icons,  given that we have e.g. the `#streamname` stream mention typeahead syntax that tokenizes them together.  I assume that's created by the 3px margin-right that you have here and above?  I think more like 0 or 1px would likely be better.

Reply:
In the issue, @rishig said that we should have more space between the global filters icons and text and less space between the `#`s and the streams. This is why I added back those 3px. 

I removed the 3px margin from both global filters and streams and also reduced the font size of:
- `All messages` icon from `17px` to `15px` and
- `Private messages` from`12px`to `11px`
For the private messages I also moved the icon 1px to the left, because it was not aligned with the others.
The screenshots are: 
- left: how it looks like with the margin and with current icon sizes
- right: without the 3px margin and with the two top icons smaller 
![Screen Shot 2019-03-24 at 19 42 02](https://user-images.githubusercontent.com/18381652/55030208-27425900-500c-11e9-9a81-a15abf9e4924.png)![Screen Shot 2019-03-26 at 21 07 34](https://user-images.githubusercontent.com/18381652/55030247-3a552900-500c-11e9-948a-fa41899c63e5.png)

I didn't push the changes yet, I'll push if you think it looks good.

Reply:
The one on the right looks great! Two thoughts:
* I could imagine dropping 1px further from each of the icons in the top left, so that they are the same size as the lock icon. 
* The new centering makes the lock icon look like it's too far to the left, relative to the `#`s?

Reply:
I updated the PR. 
- I decreased the font of the top left icons so they are the same size as the lock
![Screen Shot 2019-04-11 at 12 43 46](https://user-images.githubusercontent.com/18381652/55952255-6a7e0800-5c59-11e9-86ad-f0898a024d33.png)

- you were right @rishig that the lock was a bit to the left (the difference was less than 1 px though, and it was only on Chrome). I fixed it by changing from `0.25em` to `3px`, I am not sure why on Chrome there was a difference when using `0.25em` between the `lock` and the `#`

Here is how it looks like with these changes (on the right). The left is the version on live, just for reference.
![Screen Shot 2019-04-11 at 12 56 06](https://user-images.githubusercontent.com/18381652/55952203-51755700-5c59-11e9-9a46-0f9ee698e3a1.png)![Screen Shot 2019-04-11 at 12 15 52](https://user-images.githubusercontent.com/18381652/55952108-0a876180-5c59-11e9-9122-327854097f04.png)